### ISSUE TITLE
Files Missing？

### ISSUE THREAD

**Issue by goforher** [jishengpeng/WavTokenizer#39]
In line 15 of `ac.py`, the code `from ..binary import BitPacker, BitUnpacker` references `binary`, but there is no such folder or dependency package in the project.

**Comment by jishengpeng** [comment]
> In line 15 of `ac.py`, the code `from ..binary import BitPacker, BitUnpacker` references `binary`, but there is no such folder or dependency package in the project.

This may not seem important

**Comment by goforher** [comment]
OK, thanks

**Comment by goforher** [comment]
<img width="1098" alt="image" src="https://github.com/user-attachments/assets/5972e8c6-d031-4ebc-95d0-df35d8a94a2c">
Hello, I have a question. Why is the `forward` method in this class not using the lines of code `# residual = residual - quantized.detach()` and `# quantized_out = quantized_out + quantized`, while the `encode` and `decode` methods do? Won't this cause inconsistency between training and inference?

**Comment by jishengpeng** [comment]
> <img alt="image" width="1098" src="https://private-user-images.githubusercontent.com/76515541/370666990-5972e8c6-d031-4ebc-95d0-df35d8a94a2c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyNjY0NjEsIm5iZiI6MTcyNzI2NjE2MSwicGF0aCI6Ii83NjUxNTU0MS8zNzA2NjY5OTAtNTk3MmU4YzYtZDAzMS00ZWJjLTk1ZDAtZGYzNWQ4YTk0YTJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI1VDEyMDkyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJjNGVhNWZkYzgwMGM1M2NmNGFlNDM4MDAzZGQ0OWQ4MGQyZGRjNWE2YTU2NTcyYzZlYWQ1YjJjMTcwMGQxNGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.w6UKOhy1C6frgk8onDepweRfQpUkBP4PN_gr_EWucUw"> Hello, I have a question. Why is the `forward` method in this class not using the lines of code `# residual = residual - quantized.detach()` and `# quantized_out = quantized_out + quantized`, while the `encode` and `decode` methods do? Won't this cause inconsistency between training and inference?

The two functions are not utilized; instead, the forward function is employed.

**Comment by goforher** [comment]
Alright, thank you.

**Comment by goforher** [comment]
Hello, when I try to retrain the model using the AISHELL-3 dataset, I encounter this error as shown in the picture. The configuration file is as follows; could it be due to an incorrect configuration file or is it caused by the dataset?
<img width="1151" alt="image" src="https://github.com/user-attachments/assets/d8e10943-8e53-4995-9c06-0fdc102ae076">
<img width="1048" alt="image" src="https://github.com/user-attachments/assets/2b830ee1-7a7a-444b-93fe-321063e503cf">

<img width="702" alt="image" src="https://github.com/user-attachments/assets/5e688057-6ca2-4c1f-a8fd-65488d34ba50">
<img width="918" alt="image" src="https://github.com/user-attachments/assets/5b60b269-ea07-4ece-be0e-6b84f6effc23">
